Traceback (most recent call last):
  File "/home/rgg2706/Multimodal-Sentiment-Analysis/Models/MOASC3/run_project.py", line 4, in <module>
    from prepare_datasets import *
  File "/home/rgg2706/Multimodal-Sentiment-Analysis/Models/MOASC3/prepare_datasets.py", line 53, in <module>
    data_dir = os.path.join(datasets_path[idx], "data")  # => "/home/.../Datasets/MOA-MVSA-single/data"
                            ^^^^^^^^^^^^^
NameError: name 'datasets_path' is not defined. Did you mean: 'datasets_names'?
